src/containers/forgot-password/ForgotPasswordReset.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useParams } from 'react-router-dom';
+import { Button, Form, Input, Typography } from 'antd';
+import authClient from '../../auth/authClient';
+
+const { Title } = Typography;
+
+const ForgotPasswordReset: React.FC = () => {
+  const { key } = useParams();
+
+  // gotta change this
```

Comment:
What has to change? If you're gonna commit a comment, at least make it clear enough someone else could do the work if necessary.

Reply:
Whoops, nothing has to change. We forgot to delete that ;)